e2e/qgb/orchestrator_test.go

```diff
@@ -253,8 +252,8 @@ func TestOrchestratorWithMultipleValidators(t *testing.T) {
 		CORE3ACCOUNTADDRESS,
 	)
 	// assert the confirm exist
-	assert.NoError(t, err)
```

Comment:
super-ultra-minor-optional-can-ignore-nit: the comment `// assert` could be changed to check or smt similar

Reply:
done